test/travis.sh

```diff
@@ -42,11 +42,19 @@ set -e
 
 if ! $ONLY_RULESETS_CHANGED; then
   echo >&2 "Core code changes have been made."
+
+  if [ "$TEST" == "eslint" ]; then
+    echo >&2 "Running ESLint."
+    docker run --rm -ti -v $(pwd):/opt node bash -c "cd /opt/chromium && npm install && npm test"
+    [ `git diff --name-only | wc -l` -eq 0 ]
```

Comment:
This command does not generate a diff, it only reports errors (since the `eslint` `--fix` flag is not used).  Therefore line 48 should be sufficient, 49 can be removed.